torchbraid/torchbraid_app.pyx

```diff
@@ -161,6 +162,7 @@ class BraidApp:
     braid_SetNRelax(core,0,0) # set F relax on fine grid
     braid_SetCFactor(core,-1,self.cfactor) # -1 implies chage on all levels
     braid_SetAbsTol(core,self.abs_tol)
+    #braid_SetCRelaxWt(core, -1, 1.2)   # Turn on weighted relaxation, probably want to add command line argument
```

Comment:
What is weighted relaxation?

Reply:
This carries out weighted-Jacobi inside FCF.  It basically weight's the C-relaxation step, and turns one full FC-sweep into a weighted block-Jacobi relaxation, where the block size is the coarsening factor.

It can improve convergence (modestly), but it also allows us to smear solution values between XBraid solves and XBraid iterations.